third_party/nvfuser/csrc/scheduler/normalization.cpp

```diff
@@ -1047,11 +1328,50 @@ std::shared_ptr<ReductionParams> getPersistentHeuristics(
   // The persistent buffer is detected as being t1, which would save the
   // persistent buffer as a float, however we could obviously just save t0 which
   // is half and would take half the memory. A more complex scenario of this
-  // which requires more advanced analysis is batch norm backwards.
+  // which requires more advanced analysis is inner_batch norm backwards.
```

Comment:
Is this change intentional? I assume the original comment just meant batch normalization.

Reply:
there is an error when I batch to inner_batch! change it back to batch.